draft-ietf-quic-transport.md

```diff
@@ -1654,6 +1668,16 @@ disable_migration (0x0009):
   address other than that used to perform the handshake.  This parameter is a
   zero-length value.
 
+requested_connection_ids (0x000c):
+
+: A 32-bit unsigned integer value indicating the totoal number of connection IDs
+  the endpoint would like to maintain in active use at any given time, as
```

Comment:
Again, use in which direction? Shouldn't each side be able to say how many CIDs it wants to be able to send to, since the receiver can generate their CIDs at will?